connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TaskConfig.java

```diff
@@ -49,6 +49,6 @@ public TaskConfig() {
     }
 
     public TaskConfig(Map<String, ?> props) {
-        super(config, props);
+        super(config, props,true);
```

Comment:
Also, you're missing a space here that causes the build to fail. I can clean up on commit, but you'll want to make sure you run tests (even if just `./gradlew clean testConnect` to restrict to connect tests). Running from an IDE won't run extra stuff attached to the test targets such as checkstyle.